gitbook-docs/market/overview.md

```diff
@@ -1,63 +1,84 @@
 # Terminology
 
 {% hint style="warning" %}
-The Proof Market version represented here is beta. We expect constant changes to the below as per the feedback we receive from participants.
+The Proof Market is currently in its beta version.
+We expect there will be continuous changes in it following the feedback we receive from the customers.
 {% endhint %}
 
-## Spot vs Futures
+## Spot vs. futures
 
-Every marketplace has a set of instruments. There are buyers and sellers in the market for each of these instruments. Example: London Stock Exchange allows users to buy/sell stocks of companies, or Binance allows users to buy/sell Bitcoin- these are known as spot markets as you own the commodity.&#x20;
+Every marketplace has a set of instruments.
+The market has buyers and sellers for each of these instruments.
+For example, London Stock Exchange allows users to buy and sell stocks of companies, Binance allows users
+to buy and sell Bitcoins — these are known as spot markets because you own the commodity.
 
-Exchanges also allow buying/selling of a future price of the instrument. Example: Binance allows users to buy/sell future values of Bitcoin via perpetuals - these are known as derivative markets as you do not own the underlying instrument but are just speculating on its price.
+Exchanges also allow buying and selling a future price of the instrument, like Binance enables users
+to buy and sell future values of Bitcoin via perpetual contracts.
+These are called derivative markets, as you do not own the underlying instrument
+but are just speculating on its price.
 
-\=nil;'s proof market is more on the lines of a Spot Market rather than a speculative value of proof generation.
+`\=nil;` Proof Market is more on the lines of a Spot Market rather than a speculative
```

Comment:
why "Spot Market" and not "spot market"? It's not a proper name

Reply:
Also, why are we so uncertain here? It may be a spot market, unlikely a speculative market, probably a spot, yeah...

PM sells commodities: the proofs. @SK0M0R0H, is there any trace of speculation on the price on our Proof Market?